### DISCUSSION TITLE
How to save position between sessions

### DISCUSSION THREAD

**Discussion by jcdutton** [helix-editor/helix#4179] [Ideas]
Hi,

vim has a feature whereby you can be editing a file, exit vim, and then go back into vim and it will return you to the position you were in before.
Does hx have such a feature?
Another useful feature of vim is when searching for something with "/word"
it will highlight all the instances of "word" on the visible page, with the cursor automatically moved to the closest next one.

**Comment by the-mikedavis** [comment]
This may be covered by #401

**Comment by the-mikedavis** [comment]
Highlighting search results: https://github.com/helix-editor/helix/issues/1733

**Comment by glow12121** [comment]
@jcdutton 

> vim has a feature whereby you can be editing a file, exit vim, and then go back into vim and it will return you to the position you were in before.

What feature is it?
